vendors/espressif/boards/esp32/esp32_devkitc.cmake

```diff
@@ -10,10 +10,11 @@ afr_set_board_metadata(IS_ACTIVE "TRUE")
 afr_set_board_metadata(DATA_RAM_MEMORY "520KB")
 afr_set_board_metadata(PROGRAM_MEMORY "4MB")
 afr_set_board_metadata(CODE_SIGNER "AmazonFreeRTOS-Default")
-afr_set_board_metadata(SUPPORTED_IDE "CMakeBuildSystem")
+afr_set_board_metadata(SUPPORTED_IDE "CMakeBuildSystem;MakeBuildSystem")
 afr_set_board_metadata(RECOMMENDED_IDE "CMakeBuildSystem")
 afr_set_board_metadata(IDE_CMakeBuildSystem_NAME "CMakeBuildSystem")
 afr_set_board_metadata(IDE_CMakeBuildSystem_COMPILER "GCC")
-afr_set_board_metadata(IDE_CMakeBuildSystem_RECOMMENDED "TRUE")
 
-afr_set_board_metadata(IDE_CMakeBuildSystem_PROJECT_LOCATION "${AFR_ROOT_DIR}/vendors/espressif/boards/esp32/aws_demos/make")
\ No newline at end of file
+afr_set_board_metadata(IDE_MakeBuildSystem_PROJECT_LOCATION "${AFR_ROOT_DIR}/vendors/espressif/boards/esp32/aws_demos/make")
```

Comment:
You can use `${CMAKE_CURRENT_LIST_DIR}` instead of `${AFR_ROOT_DIR}/vendors/espressif/boards/esp32`